solarforecastarbiter/io/reference_observations/sfa_reference_sites.csv

```diff
@@ -244,3 +244,10 @@ interval_length,name,latitude,longitude,elevation,network_api_id,network_api_abb
 1,"RTC,FSEC,Baseline",28.405,-80.7709,10.27022838592529,1403,,Etc/GMT+5,https://developer.nrel.gov/docs/solar/pvdaq-v3/,NREL PVDAQ
 1,"RTC,NV,Baseline",36.0275,-114.9215,695.5903930664062,1423,,Etc/GMT+8,https://developer.nrel.gov/docs/solar/pvdaq-v3/,NREL PVDAQ
 1,"RTC,NREL,Baseline",39.7397,-105.174,1764.428344726562,1426,,Etc/GMT+7,https://developer.nrel.gov/docs/solar/pvdaq-v3/,NREL PVDAQ
+1,California ISO,37,-120,0,CISO-ALL,,America/Los_Angeles,https://www.eia.gov/opendata/,EIA
+1,Electric Realiability Council of Texas,31,-98,0,ERCO-ALL,,America/Chicago,https://www.eia.gov/opendata/,EIA
+1,ISO New England,44,-71,0,ISNE-ALL,,America/New_York,https://www.eia.gov/opendata/,EIA
+1,MISO,41,-91,0,MISO-ALL,,America/Chicago,https://www.eia.gov/opendata/,EIA
+1,NYISO,42,-75,0,NYIS-ALL,,America/New_York,https://www.eia.gov/opendata/,EIA
+1,PJM,39,-79,0,PJM-ALL,,America/New_York,https://www.eia.gov/opendata/,EIA
+1,Southwest Power Pool,40,-99,0,SWPP-ALL,,America/Chicago,https://www.eia.gov/opendata/,EIA
```

Comment:
Do we want to use these DST aware timezones or the non-DST Etc/GMT equivalents? I think either would work and it's just a matter of the default report formatting.

Reply:
I'm open to either. I have a slight preference for the DST-aware timezones since ISOs/RTOs tend to report data in DST-aware timezones on their own websites, but I defer to you for a final decision.

Reply:
sure, let's go with DST-aware zones.